portfolio/src/main/webapp/index.html

```diff
@@ -6,70 +6,24 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="name">
-      <h1>George Zheng</h1>
-    </div>
-    <hr>
-    <button type="button" onclick="deleteComments()">Delete Comments</button>
-    <ul id="comments-history"></ul>
-    <form>
-      <p>Enter number of comments to view:</p>
-      <input type="number" min="1" value="1" name="lim">
-      <br/><br/>
-      <button type="button" onclick="getComments(lim.value)">View Comments</button>
-    </form>
-    <form action="/comments" method="POST">
-      <p>Add a new comment:</p>
-      <input type="text" name="comment">
-      <br/><br/>
-      <input type="submit" />
-    </form>
-    <div id="background">
-      <p>Hi there! I'm George Zheng, and this is my portfolio website you're currently on.
-      <p>I was born on October 25th, 2000 in the borough of Manhattan in New York City. I grew up in that same city, attending Stuyvesant High School until recently. Currently, I am a rising junior at the Massachusetts Institute of Technology, pursuing a degree in Computer Science and Engineering. 
-    </div>
-    <div id="background">
-      <p>I have worked on a variety of projects spanning several academic disciplines (check out some selected works featured below!). In my free time, I enjoy playing video games (most recently, League of Legends and Realm of the Mad God), watching anime, and napping. 
+  <body onload="handleLogin()">
+    <div id="commenting">
+      <button type="button" onclick="deleteComments()">Delete Comments</button>
+      <ul id="comments-history"></ul>
       <form>
-        <input type="button" value="Select a project!" onClick="chooseFeaturedProj()">
+        <p>Enter number of comments to view:</p>
```

Comment:
Consider making it a `<label>` that is associated with the `<input>` below.

FYI, https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
changed